R/equivalence_test.R

```diff
@@ -29,14 +29,19 @@
 #' }
 #'
 #' @export
-equivalence_test <- function(posterior, bounds = "default", ci = .95, verbose = TRUE) {
+equivalence_test <- function(posterior, range = "default", ci = .95, verbose = TRUE) {
   UseMethod("equivalence_test")
 }
 
 
 #' @export
-equivalence_test.numeric <- function(posterior, bounds = "default", ci = .95, verbose = TRUE) {
-  out <- as.data.frame(rope(posterior, bounds = bounds, ci = ci))
+equivalence_test.numeric <- function(posterior, range = "default", ci = .95, verbose = TRUE) {
+  if (length(ci) > 1) {
+    stop("'ci' needs be a single value.", call. = FALSE)
+  }
```

Comment:
@DominiqueMakowski Line 46 gives a warning when CI is longer than 1 e.g. `ci = c(.6, .8)`. This will not pass through CRAN checks, that's why I wanted to make users aware that only length-one is allowed for `ci`. However, I just saw in the examples and tests, that for numeric vectors, `equivalence_test()` was intended by you to work with multiple `ci`-values. How to deal with this?

Reply:
Ok, this one is actually resolved by this commit: https://github.com/easystats/bayestestR/pull/29/commits/1eaac43fa2e9f857dae4dcd9065e2e0a7f98dee0

I need to revise the `print()`-method to print out the necessary information for equivalence-tests with multiple CI.